CMakeLists.txt

```diff
@@ -42,10 +44,10 @@ message(STATUS "Build date set to ${BUILD_DATE}.")
 include(GrComponent)
 
 # Set the version information here
-SET(VERSION_MAJOR 3)
-SET(VERSION_API   10)
-SET(VERSION_ABI   0)
-SET(VERSION_PATCH 0-rc2)
+set(VERSION_MAJOR 3)
+set(VERSION_API 10)
```

Comment:
I actually preferred the old indenting here.

Reply:
agreed, don't know whether cmake-lang supports that, though. Haven't found an option, so [asking on SO](https://stackoverflow.com/questions/70597446/keep-multiple-space-aligned-setvariable-value-the-way-they-were-during-cmake).

Reply:
Isn't this all you need? https://cmake-format.readthedocs.io/en/stable/format-features.html?highlight=disable#disable-formatting-locally

Reply:
Yep! As [someone helpfully pointed out as well](https://stackoverflow.com/questions/70597446/keep-multiple-space-aligned-setvariable-value-the-way-they-were-during-cmake/70846835#70846835) :)